autogpt/config/ai_config.py

```diff
@@ -6,6 +6,7 @@
 
 import os
 from pathlib import Path
+from platform import platform
```

Comment:
```python
>>> from platform import platform
>>> platform.system()
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: 'function' object has no attribute 'system'
```

Reply:
`import platform` was working fine but the linter check was throwing an error, thanks for pointing it out I will look into it